src/app/components/wallet-deeplink/wallet-deeplink.component.ts

```diff
@@ -83,10 +84,38 @@ export class WalletDeeplinkComponent {
   constructor(private sanitizer: DomSanitizer, private cd: ChangeDetectorRef) {}
 
   private setLinks(): void {
+    this.bloomDeepLink = this.getBloomDeepLink();
     this.fireflyDeepLink = this.getFireflyDeepLink();
     this.tanglePayDeepLink = this.getTanglePayDeepLink();
   }
 
+  private getBloomDeepLink(): SafeUrl {
+    if (!this._network || this._network === Network.IOTA || this._network === Network.ATOI) {
+      // TODO: IOTA network support in Bloom
```

Comment:
When do you plan to support this? I think we might need to add logic to hide the button if IOTA network is used. Otherwise it'll confuse users.

Reply:
We want to release an early access without IOTA support and then start working asap on IOTA support.

Reply:
I can hide if there is no deep link though.

Reply:
Yeah that would make sense. Otherwise it will confuse users. Especially they might start to think it does not work at all.